addons/print/wizard/print_order_sendnow_wizard.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from openerp import api, fields, models
+
+
+class print_order_sendnow_wizard(models.TransientModel):
+
+    _name = 'print.order.sendnow.wizard'
+    _rec_name = 'create_date'
+
+    # --------------------------------------------------
+    # MODEL FIELDS
+    # --------------------------------------------------
+    print_order_sendnow_line_wizard_ids = fields.One2many('print.order.sendnow.line.wizard', 'print_order_sendnow_wizard_id', string='Lines')
+
+    # --------------------------------------------------
+    # METHODS
+    # --------------------------------------------------
+    @api.model
+    def default_get(self, fields):
+        """ create the lines on the wizard """
+        res = super(print_order_sendnow_wizard, self).default_get(fields)
+        active_ids = self._context.get('active_ids', [])
+        if active_ids:
+            # create order lines
+            lines = []
+            for rec in self.env['print.order'].browse(active_ids):
+                if rec.state != 'sent':
+                    lines.append((0, 0, {
+                        'res_model': rec.res_model,
+                        'ink' : rec.ink,
+                        'partner_id' : rec.partner_id.id,
+                        'provider_id' : rec.provider_id.id,
+                        'state' : rec.state,
+                        'order_id' : rec.id
+                    }))
+            res['print_order_sendnow_line_wizard_ids'] = lines
+        return res
+
+    @api.multi
+    def action_apply(self):
+        Print_order = self.env['print.order']
+        for wizard in self:
+            order_ids = [line.order_id.id for line in wizard.print_order_sendnow_line_wizard_ids]
+            Print_order.process_order_queue(order_ids)
+        return {'type': 'ir.actions.act_window_close'}
+
+
+
+class print_order_sendnow_line_wizard(models.TransientModel):
```

Comment:
Actually, what is the purpose of this model ? Above you seem to use only line.order_id ... so maybe you could directly link some print order in the sendnow wizard.

The issue will probably be the reverse many2one field... so maybe having a computed field could solve this issue. To check I think.